Makefile

```diff
@@ -190,17 +190,25 @@ konduit-cleanup:
 	sed -i '' -e '/url\: "postgres/d' config/database.yml; \
 	exit 0
 
-# Creates a konduit to the snapshot DB and points development to
-# it. The konduit URL is removed when the konduit is closed.
-konduit-snapshot: get-cluster-credentials
+define KONDUIT_CONNECT
 	trap 'make konduit-cleanup' INT; \
 	tmp_file=$$(mktemp); \
 	$(MAKE) konduit-cleanup; \
 	{ \
-			(tail -f -n0 "$$tmp_file" & ) | grep -q "postgres://"; \
-			postgres_url=$$(grep -o 'postgres://[^ ]*' "$$tmp_file"); \
-			echo "$$postgres_url"; \
-			sed -i '' -e "s|npq_registration_development|&\\n  url: \"$$postgres_url\"|g" config/database.yml; \
+		(tail -f -n0 "$$tmp_file" & ) | grep -q "postgres://"; \
+		postgres_url=$$(grep -o 'postgres://[^ ]*' "$$tmp_file"); \
+		echo "$$postgres_url"; \
+		sed -i '' -e "s|npq_registration_development|&\\n  url: \"$$postgres_url\"|g" config/database.yml; \
 	} & \
-	bin/konduit.sh -d s189p01-cpdnpq-pd-pg-snapshot -k s189p01-cpdnpq-pd-app-kv npq-registration-production-web -- psql > "$$tmp_file"
+	bin/konduit.sh -d
+endef
+
+# Creates a konduit to the snapshot DB and points development to it. The konduit URL is removed when the konduit is closed.
```

Comment:
unless its a huge faff because of `snapshot`, maybe we can have one command and add extra envs we need to use to the `_aks` example `migration_aks` config files to refer to them as required and build the setup, as an example from above `${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-rg`, unless you already tried :)

Reply:
Oh I see what you mean. Yeah that makes sense, although I think we'll still need two commands but at least we won't need to add more if we want to konduit to a new environment. Something like this, I'll test it tomorrow morning and commit it if it works:

```
# Creates a konduit to the DB and points development to it. The konduit URL is removed when the konduit is closed.
konduit: get-cluster-credentials
	$(KONDUIT_CONNECT) ${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-pg -k ${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-app-kv cpd-ecf-${APP_ID}-web -- psql > "$$tmp_file"
	exit 0

# Creates a konduit to the snapshot DB and points development to it. The konduit URL is removed when the konduit is closed.
konduit-snapshot: get-cluster-credentials
	$(KONDUIT_CONNECT) ${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-pg-snapshot -k ${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-app-kv cpd-ecf-${APP_ID}-web -- psql > "$$tmp_file"
	exit 0

```